pytorch_lightning/utilities/__init__.py

```diff
@@ -34,14 +34,20 @@ def _module_available(module_path: str) -> bool:
     >>> _module_available('bla.bla')
     False
     """
-    mods = module_path.split('.')
-    assert mods, 'nothing given to test'
-    # it has to be tested as per partets
-    for i in range(len(mods)):
-        module_path = '.'.join(mods[:i + 1])
-        if importlib.util.find_spec(module_path) is None:
-            return False
-    return True
+    # todo: find a better way than try / except
+    try:
+        mods = module_path.split('.')
+        assert mods, 'nothing given to test'
+        # it has to be tested as per partets
+        for i in range(len(mods)):
+            module_path = '.'.join(mods[:i + 1])
+            if importlib.util.find_spec(module_path) is None:
+                return False
+        return True
+    except ImportError:
+        return False
```

Comment:
How can this throw an `ImportError`? isn't that avoided by using `find_spec`?

Reply:
You are right ! Removed.